ToolChain/Assembler/Parser/Parser.py

```diff
@@ -187,18 +186,15 @@ def _parseCodeLine(self, line: str=None):
         possibleCodes = None
 
         # We evaluate the line and extract any bits of relevant info to build the appropriate instruction
-        self._evaluateAndExtractInfoFromLine(line, buildInstruction)
+        self._evaluateAndExtractInfoFromLine(line, buildInstruction, raw_line)
```

Comment:
I think raw_line should be renamed to rawLine as all variable in spartacus are camelCase

Reply:
@kommisar5150 can you fix this please?